srtcore/list.cpp

```diff
@@ -156,21 +161,36 @@ int CSndLossList::insert(int32_t seqno1, int32_t seqno2)
             i = m_iHead;
 
          while ((-1 != m_caSeq[i].next) && (CSeqNo::seqcmp(m_caSeq[m_caSeq[i].next].data1, seqno1) < 0))
-            i = m_caSeq[i].next;
+         {
+            if (i == m_caSeq[i].next)
+               m_caSeq[i].next = -1;
+            else
+               i = m_caSeq[i].next;
+         }
```

Comment:
An already quite poor readability of this original loop has deteriorated through this change. As previously it looked like jumping to the next element, this time it's even barely possible to determine if this loop has an end condition at all. Please reform to the standard-looking range-searching loop.